src/main/java/com/blackducksoftware/integration/hub/HubIntRestService.java

```diff
@@ -375,15 +253,15 @@ public ProjectItem getProject(final String projectUrl) throws IOException, BDRes
 
 	}
 
-	public ReleaseItem getProjectVersion(final String versionUrl)
-			throws IOException, BDRestException, URISyntaxException {
-		final ClientResource resource = createClientResource(versionUrl);
+	public ReleaseItem getProjectVersion(final String versionUrl) throws IOException, BDRestException,
+	URISyntaxException {
+		final ClientResource resource = getRestConnection().createClientResource(versionUrl);
```

Comment:
I agree that it makes sense to add methods to RestConnection for PUT, POST, DELETE. I was thinking it made sense to do that as the need for them arose.

Does RestConnection.httpGetFromAbsoluteUrl() not serve the purpose for submitting Get requests?